nebula-logger/core/main/logger-engine/classes/Logger.cls

```diff
@@ -2428,6 +2428,211 @@ global with sharing class Logger {
         return finest().setMessage(message);
     }
 
+    /**
+     * @description Creates a log entry for any results within the provided List DeleteResult> where isSuccess() == true
+     * @param  loggingLevel  The logging level to use for the log entry
```

Comment:
!= true, right?

Reply:
Whooops! Yup, I'll fix these comments + regenerate the docs files. Thanks for catching this!